### PR TITLE
feat(quill): handle drafts

### PR DESCRIPTION
**Pull request by phatv**
Also fixed a couple bugs from reply/edit message

### CODE REVIEW

File: src/components/ComposerQuill/composer.js [hunk 1]
```diff
@@ -111,15 +127,24 @@ class Composer extends React.Component {
...
+      this.saveToDraft();
     }
+  }
+
+  saveToDraft() {
+    const {draft} = this.props;
 
     if (draft?.save) {
-      draft.save(this.quill.root.textContent, draft.id);
+      draft.save(this.quill.getText().trim(), draft.id);
```
Comment (chrisadubois) [inline src/components/ComposerQuill/composer.js]: does .getText() ever return falsy?

File: src/components/ComposerQuill/composer.js [hunk 2]
```diff
@@ -62,10 +64,13 @@ class Composer extends React.Component {
...
     });
 
-    // inserts text into composer
+    // inserts the initial text to the composer
+    // may contain formats as html tags, so convert those to markdowns
     if (draft?.value) {
-      // converts text from html to markdown
-      const text = td.turndown(draft.value);
+      // replace new lines with <br> tag so turndown can convert it properly
+      const modified = draft.value.replace(/\n/g, '<br />');
```
Comment (chrisadubois) [inline src/components/ComposerQuill/composer.js]: oof wysiwig

Comment (phatv) [reply src/components/ComposerQuill/composer.js]: you alright there?

File: src/components/ComposerQuill/composer.js (hunk 1 above)
Comment (phatv) [reply src/components/ComposerQuill/composer.js]: it returns the string in the composer, which could be `''`

**Pull request merged**